docs/development/i18n.rst

```diff
@@ -64,6 +59,19 @@ files into the corresponding ``*.j2`` file and committed to the
 SecureDrop repository. They are then installed when configuring Tails
 with the ``tasks/create_desktop_shortcuts.yml`` tasks.
 
+We don't expect translators to deal with all these files directly, of
+course. Translation happens on our `Weblate`_ server, which is
+configured to use a fork of the main `SecureDrop git repository`_.
+
+At the start of the the release process, the localization manager
```

Comment:
a link to the release process docs, which I think also defines the concept of a localization manager, might be good here.